frontend/public/components/edit-yaml.jsx

```diff
@@ -64,19 +66,40 @@ const WithYamlTemplates = (Component) =>
     );
   };
 
+const getModel = (models, obj) => {
```

Comment:
If you make this part of the component class, you wouldn't need to pass `models` as an argument since it would be available in props.

Reply:
Yeah, I changed this such that they are both class methods where the createResources is passed to the new results component as a props, and I don't have to shuffle models around anymore.